### PR TITLE
FizzBuzz in Python3

### PR DESCRIPTION
**Pull request by g3wanghc**

### CODE REVIEW

File: fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def main():
+    print ([fizz_buzz(n) for n in range(1, 100)])
```
Comment (bjdixon) [inline fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py:2]: nice list comprehension :sunglasses:

File: fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+def fizz_buzz(n):
```
Comment (bjdixon) [inline fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py:5]: PEP8 suggests surrounding top level functions with 2 line breaks.

File: fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	if (n % 15 == 0):
+		return 'FizzBuzz'
```
Comment (bjdixon) [inline fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py]: Should probably avoid mixing tabs and spaces. Also your tabs are quite large. I think PEP8 would suggest these tabs should be replaced with 4 spaces

File: fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+	if (n % 5 == 0):
+		return 'Fizz'
+	if (n % 3 == 0):
+		return 'Buzz'
```
Comment (bjdixon) [inline fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py]: The convention is usually `n % 3 == 'Fizz'` and `n % 5  == 'Buzz'` but I guess it doesn't really matter.

**Comment by g3wanghc** [comment]
@bjdixon Done. 😎

**Pull request merged**